### ISSUE TITLE
Track inviters and ban-ners

### ISSUE THREAD

**Issue by staltz** [ssb-ngi-pointer/go-ssb-room#217]
This is an idea for the future, not something for [community launch](https://github.com/ssb-ngi-pointer/go-ssb-room/milestone/2):

**Track inviter:** it would be good that for each member, we would store and display the ID of the other member who invited them. If things start going wrong in the community up to the point where Wario has to be removed as a member, it would be good to know who invited Wario to begin with. Was it Goomba? And who invited Goomba? And so forth.

**Track ban-ner:** similarly, we could track who banned someone.

**Comment by cryptix** [comment]
For invites we already track this.

For banned people we need to change the schema a bit (right now we just have the comment) and need to think about if we should cascade deletes when a the ban-ner is removed.

**Comment by staltz** [comment]
> For invites we already track this so we need to copy this info to the member and change the schema a bit.
> 
> For banned people we need to change the schema a bit (right now we just have the comment).

:+1:

> For both changes we need to think about how we should cascade deletes when a the ban-ner/inviter is removed. I suspect we want to update to _unknown_?

Good point! Could we keep the SSB ID? Then the community can still track what happened.

I guess in the database that would mean storing both the Member ID and the SSB ID of the inviter, because the Member ID allows the UI to show a link to the inviter, and the SSB ID is a fallback when there is no longer a Member ID.

**Comment by staltz** [comment]
Good point, I can see that an audit table and an audit page which just lists the latest actions can be sufficient for the purpose of tracking who invited who. It also would provide a bit more time-related context.

One note though https://github.com/ssb-ngi-pointer/rooms2/issues/8 is about an audit *feed*, I think before we have that, it would be useful to have an audit *table*.
